FoodOrderingSystem.py

```diff
@@ -0,0 +1,884 @@
+        session = self.Session()
+        view_order_item = self.customer.view_order_per_item(order_id, session)
+        if view_order_item:
+            for fc, fd, cos in view_order_item:
+                print("\nFood category: {} \nFood name: {} \nFood price: {} \nFood quantity: {} \nTotal per item: {}".format(fc.name, fd.food_name, fd.price, cos.food_qty, (fd.price*cos.food_qty)))
+                        
+        view_status = self.customer.view_orders_status(order_id, session)
+        if view_status:
+            for cos, cd, cosa, dp in view_status:                        
+                print("\nCustomer name: {} \nOrder ID: {} \nDeliver person name: {} \nOrder status: {} \nTotal price: {}".format(cd.cust_name, cosa.order_id, dp.delivery_person_name, cosa.order_status, cosa.bill_amount))
+
+class DeliveryPerson(Base, Employee):
+    """ Represents delivery person """
+
+    __tablename__ = 'delivery_person'
+
+    delivery_person_id = Column(Integer(), primary_key=True, nullable=False, unique=True, autoincrement=True)
+    delivery_person_name = Column(String(30), nullable=False)
+    delivery_person_phone = Column(Integer(), nullable=False)
+
+    def update_order(self, order_id, order_status, session):
+        """ Delivery person can update the order """
+
+        super(DeliveryPerson, self).__init__()
+
+        update = session.query(
+            CustOrderStatus
+            ).filter(CustOrderStatus.order_id == order_id
+            ).update(
+                {CustOrderStatus.order_status: order_status}, 
+                synchronize_session=False
+                )
+        try:
+            session.commit()
+            return update
+        except:
+            session.rollback()
+            raise Exception("update not completed!")
+        finally:
+            session.expunge_all()
+            session.close()
+
+def main():
+    """ The user interface in which employee/customer will use to perform actions on the Food Ordering System. """
+
+    db_url = "sqlite:///fos.db"
+    print("Connecting to {}".format(db_url))
+
+    fos = Controller(db_url)
+    fos.bootstrap()
+
+    welcome_message = """ 
+    Welcome to the Food Ordering System! Please press the below options:
+
+    1. Employee
+    2. Customer
+
+    Your Option:
+
+    """
+
+    option = int(input(welcome_message))
+
+    if option == 1:
+        emp_options = """         
+        Welcome to the employee interface! Please press the below options:
+        
+        0. Quit
+        1. Add food category
+        2. Add food details
+        3. Add delivery person
+        4. Assign delivery person to order 
+        5. Update order (Only for delivery person use)
+        6. View order details
+        7. View order status
+        8. View revenue today
+        9. Delete order
+    
+        Your Option:
+
+        """
+
+        employee_options = int(input(emp_options))
+
+        while employee_options != 0:
+
+            if employee_options == 1:
```

Comment:
For readibility, use constants like this:

OPTION_ADD_FOOD_CATEGORY = 1
OPTION_ADD_FOOD_DETAILS = 2
...
... (etc)

And then you can put these into the `if` conditions like this:

```
if employee_options == OPTION_ADD_FOOD_CATEGORY:
  ...
  ...
elif employee_options == OPTION_ADD_FOOD_DETAILS:
  .. 
  ..
```

Reply:
Used constants